src/System.Buffers.Primitives/System/Buffers/OwnedArray.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+using System.Runtime;
+using System.Runtime.InteropServices;
+using System.Threading;
+
+namespace System.Buffers
+{
+    public class OwnedArray<T> : OwnedBuffer<T>
```

Comment:
Is the only change here moving OwnedArrray from System/Buffers/Internals to System/Buffers or are there other changes as well?